.github/workflows/unittest_lint.yml

```diff
@@ -0,0 +1,104 @@
+name: "Run lint, unit tests, and cover"
```

Comment:
The current name looks good.

Other naming ideas:
"GH actions presubmit"
or
"Lint / Unit Tests / Coverage"
?

Reply:
@tmatsuo I've added running `mypy` as well, and retitled it `Lint / Unit Tests / Coverage / Mypy`.